infra/src/main/kotlin/fi/fta/geoviite/infra/util/FreeText.kt

```diff
@@ -45,4 +49,8 @@ data class FreeTextWithNewLines private constructor(private val value: String) :
     override fun compareTo(other: FreeTextWithNewLines): Int = value.compareTo(other.value)
 
     operator fun plus(addition: String) = FreeTextWithNewLines("$value$addition")
+
+    fun escapeNewLines(): FreeText {
+        return FreeText(UnsafeString(value.replace(NEW_LINE_CHARACTER, ESCAPED_NEW_LINE)))
```

Comment:
En halunnut luoda constructoria suoraan FreeTextWithNewLinesista tai Stringistä tuolle FreeTextille niin meni nyt näin 🤔. Tässähän käy vähän tuplasanitaatiohimmeliä nyt, kun FreeTextWithNewLines on jo luotu, mutta FreeTextin rakentaja sanitoi asioita uudelleen. Parannusehdotuksia?

Reply:
Itse näen tämän kaikkein eniten string-formatting-juttuna (tässä halutaan ulostaa tämän sisältö merkkijonona, mutta siten että rivinvaihdoille on tehty temput,) en niinkään transformaationa uuteen sanitoituun instanssiin. Mitä jos tämä palauttaisikin suoraan vain stringin ja olisi nimeltään jotain tyyliin `fun toStringWithoutNewlines()`?

Reply:
Joo voisi toimia tietenkin noinkin 🤔, hmmhm

Reply:
Joo ihan validi varmaan tuo Juusen ehdotus, :+1: sille. Ja vaikka tämän pitäisi niin eikös tuo UnsafeString ole ihan turha väliolio luoda tuossa?

Reply:
FreeTextillä ei ole suorasta Stringistä rakentajaa, joten sitä varten tuo typerä väliolio tuossa majailee (en halunnut lisäillä tuota rakentajaa).

Juusen ehdotus on tosiaan varmaan se parempi ratkaisu, vaihtelen hieman myöhemmin 👍